test/unit/workDiv/src/WorkDivHelpersTest.cpp

```diff
@@ -139,3 +139,46 @@ TEMPLATE_LIST_TEST_CASE("isValidWorkDiv", "[workDiv]", alpaka::test::TestAccs)
     REQUIRE(alpaka::isValidWorkDiv(alpaka::getAccDevProps<Acc>(dev), workDiv));
     REQUIRE(alpaka::isValidWorkDiv<Acc>(dev, workDiv));
 }
+
+//! Test the constructors of WorkDivMembers using 3D extent, 3D extent with zero elements and 2D extents
+TEMPLATE_LIST_TEST_CASE("WorkDivMembers", "[workDiv]", alpaka::test::TestAccs)
```

Comment:
```suggestion
TEST_CASE("WorkDivMembers", "[workDiv]")
```
This test does not depend on any accelerator, so no need to run it multiple times.